dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -164,3 +300,138 @@ def dpnp_eigh(a, UPLO):
         ht_copy_ev.wait()
 
         return w, out_v
+
+
+def dpnp_solve(a, b):
+    """
+    dpnp_solve(a, b)
+
+    Return the solution to the system of linear equations with
+    a square coefficient matrix `a` and multiple dependent variables
+    array `b`.
+
+    """
+
+    a_usm_arr = dpnp.get_usm_ndarray(a)
+    b_usm_arr = dpnp.get_usm_ndarray(b)
+
+    b_order = "C" if b.flags.c_contiguous else "F"
+    a_shape = a.shape
+    b_shape = b.shape
+
+    res_usm_type, exec_q = get_usm_allocations([a, b])
+    if exec_q is None:
+        raise ValueError(
+            "Execution placement can not be unambiguously inferred "
+            "from input arguments."
```

Comment:
Actually this never happens, since both `a` and `b` have `sycl_queue` attribute and `get_usm_allocations` will never return `None` in that case.

Reply:
Yes, you're right. 
This check was added when I used `dpctl.utils.get_execution_queue`
Now It`s not relevant. I removed it.